gis/saga/Portfile

```diff
@@ -101,14 +101,6 @@ variant vigra description "Add VIGRA support" {
         }
     }
 
-    if {![catch {set result [active_variants hdf5 openmpi-devel]}]} {
-        if {$result} {
-            if {![variant_isset openmpi-devel]} {
-                return -code error "HDF5 has openmpi_devel enabled, please use the same variant."
-            }
-        }
-    }
-
```

Comment:
why are this and the other check not needed anymore?

Reply:
There are no more any `openmpi-devel` or `mpich-devel`. I reused this code block taken from another Portfile in https://github.com/macports/macports-ports/commit/7f8424a5f267454af77ecfcef746a42ba3aefd53, but later learned they don't exists anymore, thus were unneeded from the start.